mibitracker/tests/test_request_helpers.py

```diff
@@ -39,6 +39,36 @@ def setUp(self):
     def tearDown(self):
         self.mock_auth.stop()
 
+    def test_initializing_with_token(self):
+        fake_token = """
+            eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c
+        """
+        try:
+            request_helpers.MibiRequests(
+                'https://mibitracker-instance.ionpath.com',
+                None,
+                None,
+                fake_token
+            )
+        except ValueError as e:
+            self.fail(e)
+
+    def test_parameter_validation(self):
+        try:
+            request_helpers.MibiRequests(
+                'https://mibitracker-instance.ionpath.com',
+                None,
+                None,
+                None
+            )
+        except ValueError:
```

Comment:
Thanks for adding these tests! I'd typically use the pattern below, but what you did also makes sense and I wasn't familiar with the `fail` method and will be sure to find it useful.
```
with self.assertRaises(ValueError):
    request_helper.MibiRequests(...)
```